market/connector/docusign/README.md

```diff
@@ -38,4 +38,30 @@ Before any signing interactions between the Axon.ivy Engine and the DocuSign eSi
    Your setup was correct, if you are being asked to authorize yourself with a docuSign account.
    ![docusign-auth](doc/images/docuSign_auth.png)
 
+### Allow System Authentication (JWT)
 
+The Demo process contains a final service part, where the Axon.ivy platform acts in the name of a user.
+![docusign-props](doc/images/systemDrivenProcess.png)
+
+This interaction requires a JWT authentication setup:
+
+1. Edit the docuSign `application` as in step 3. of the general setup.
+1. In the section `Authentication` click on `Generate RSA` in order to create a secure key-pair.
+ ![docusign-gen-rsa](doc/images/authenticationGenerateRSA.png)
+
+1. Store the generated private key:
+	1. Copy the generated 'Private Key' to your clipboard
+	1. Save the changed application settings
+	1. Create a new empty text file called `docusign.pem` in your Designer 'configuration' directory
+	1. Paste the contents of your clipboard into the `docusign.pem` file
+	1. You can use another storage location for the pem file. Adjust the Rest Client Property `AUTH.systemKeyFile` to refere it. It should be a relative path to the 'configuration' directory or an absolute path on your system.
```

Comment:
Maybe mark this step as (Optional) ?

Reply:
Yes the whole section is kind of optional. That's why it already has it's own section header. But I agree that it should be more obvious that one doesn't really need it for a very simple demonstration.